webcheck.go

```diff
@@ -96,7 +104,13 @@ func (fm *Frontman) runWebCheck(data WebCheckData) (map[string]interface{}, erro
 	m := make(map[string]interface{})
 	m[prefix+"success"] = 0
 
-	httpClientWithMaxRedirects := fm.newHTTPClientWithCustomMaxRedirectsLimit(fm.Config.HTTPCheckMaxRedirects)
+	var httpTransport *http.Transport
+	if data.IgnoreSSLErrors {
+		httpTransport = fm.transportWithInsecureSSL()
```

Comment:
Why do we need a copy of `http.Transport`?
Can't we just update the SSL settings as we need them?

Reply:
> Why do we need a copy of `http.Transport`?
> Can't we just update the SSL settings as we need them?

I didn't want to put mutex here, as this would prevent doing multiple checks in parallel.

Reply:
Actually, I will create new, because it needs a deep copy otherwise (CI warns about mutex copy).